pkg/cmd/openshift-controller-manager/controller/apps.go

```diff
@@ -50,7 +50,7 @@ func RunDeploymentConfigController(ctx ControllerContext) (bool, error) {
 		return true, err
 	}
 
-	groupVersion := schema.GroupVersion{Group: "", Version: "v1"}
+	groupVersion := schema.GroupVersion{Group: "apps.openshift.io", Version: "v1"}
 	annotationCodec := legacyscheme.Codecs.LegacyCodec(groupVersion)
```

Comment:
does this prevent us from decoding existing groupless annotations?

Reply:
talked offline... this controls the controller write path. the deployer pod needs to be fixed to be able to decode existing groupless annotations

Reply:
> does this prevent us from decoding existing groupless annotations?

What do you know, this one is used for decoding.  Can I get a followup instead of a fix?  I want to stop passing it entirely.

Reply:
> What do you know, this one is used for decoding. Can I get a followup instead of a fix? I want to stop passing it entirely.

as long as the follow-up is clearly release blocking, ok